content/docs/04.5-devx/05-app-profile.md

```diff
@@ -18,21 +18,105 @@ import Tooltip from "shared/components/ui/Tooltip";
+Several Services are available to build App Profiles.
 
-<br />
+# Services
 
-- [Create an App Profile](/devx/app-profile/create-app-profile)
-- [Container Deployment](/devx/app-profile/container-deployment)
-- [App Profile Macros](/devx/app-profile/app-profile-macros)
-- [App Profile Cloning](/devx/app-profile/app-profile-cloning)
-- [App Profile Versioning](/devx/app-profile/versioning-app-profile)
+These basic Services are available:
+
+- **Container Deployment**: [Containers](https://www.docker.com/resources/what-container/) are methods of building, packaging, and deploying an application. A container includes the code, run-time, libraries, and all the dependencies required by a containerized workload. Containers are deployed to their target environment. For steps on how to deploy a container in Palette, refer to [Container Deployment](/devx/app-profile/container-deployment).
 
+- **Helm**: Palette provides out-of-box Helm registries and allows you to add registries. For more information, visit [Palette Helm Registry](/devx/registries/helm-registry#palettehelmregistry).
 
+- **Manifest**: You can construct App Profile layers using raw manifests to provision Kubernetes resources that are unavailable in Palette or Helm Charts. Pack Manifests provide a pass-through mechanism to orchestrate Kubernetes resources in a cluster. For example, specific integrations may require creation of Secrets or Custom Resource Definitions (CRDs). To achieve this, you can attach a Manifest file to the layer.
 
+<InfoBox>
+We encourage you to securely distribute credentials using secrets. You can achieve this by constructing manifests to consume parameters that use references. You can also add a manifest to create your Kubernetes secrets using macros to inject the values. In the App layer, set the environment variables using a Secret reference.
 
+**Example**
 
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+  namespace: {{.spectro.system.apptier.NAMESPACE}}
+data:
+  
+  namespace: "{{.spectro.system.apptier.NAMESPACE}}" # Resolves to a value of the variable defined in App Profile tier parameters.yaml file.
+
+  password: "{{.spectro.app.$appDepName-mongodb.PASSWORD}}" # To refer the tier output variables of the top tiers.
 	
+```
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: myweb
+  namespace: mywebnamespace
+  ...
+spec:
+  template:
+  ...
+      containers:
+        - env:
+            - name: PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: mysecret
+                  key: password
+```
+
+</InfoBox>
+
+<br />
+
+<WarningBox>
+    When adding services to App Profiles using manifest files, specify a namespace name to which the resource belongs.
+</WarningBox> 
+
+## Messaging System Services
+
+These Messaging System services are available for client-server data exchange and monitoring:
+
+- **NATS**
+- **Kafka**
+
```

Comment:
Add one liner about Kafka

Reply:
@tiwarisumit3 Instead of one-liners, I'll add links to these so users can read more if they want.

Reply:
decided against links.